test/remote/gateways/remote_clearhaus_test.rb

```diff
@@ -44,7 +44,7 @@ def test_unsuccessful_signing_request
     assert gateway.options[:private_key]
     assert auth = gateway.authorize(@amount, @credit_card, @options)
     assert_failure auth
-    assert_equal 'Neither PUB key nor PRIV key: not enough data', auth.message
+    assert_equal 'Neither PUB key nor PRIV key: unsupported', auth.message
```

Comment:
This is a new error message based on the openssl updates from the new JOSE version.